velox/exec/HashAggregation.h

```diff
@@ -50,6 +50,10 @@ class HashAggregation : public Operator {
 
   void close() override;
 
+  const std::vector<IdentityProjection>& identityProjections() const override {
```

Comment:
Please, provide an analysis of why this is correct.

Reply:
It has been removed.